lib/scheduler.pm

```diff
@@ -51,17 +51,29 @@ sub parse_schedule {
     my ($schedule) = shift;
     my @scheduled;
     for my $module (@{$schedule->{schedule}}) {
-        push(@scheduled, $module) && next unless ($module =~ s/\{\{(.*)\}\}/$1/);
+        push(@scheduled, parse_schedule_module($schedule, $module));
```

Comment:
Please add to the documentation: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/declarative-schedule-doc.md that now nested conditions are possible

Reply:
And also adapt `tools/detect_nonexistent_modules_in_yaml_schedule`

Reply:
Done